cosmo_tester/framework/test_hosts.py

```diff
@@ -432,7 +453,7 @@ def _create_config_file(self):
         config_file.write_text(install_config_str)
         return config_file
 
-    def bootstrap(self):
+    def bootstrap(self, db=False):
```

Comment:
What "db" means?

Reply:
Whether or not this is an external DB machine (if it is we do not need to enter sanity mode)

Reply:
so maybe change it to "with_db" or "db_contained"